tls/s2n_quic_support.c

```diff
@@ -42,6 +42,11 @@ int s2n_config_enable_quic(struct s2n_config *config)
     return S2N_SUCCESS;
 }
 
+bool s2n_connection_is_quic_enabled(struct s2n_connection *conn)
```

Comment:
good call on the function. makes it easy to add additional logic in the future too